client/pages/index.js

```diff
@@ -25,8 +25,10 @@ const mapStateToProps = (state ) => {
   return {};
 }
 
-const mapDispatchToProps =(dispatch)=>  {
-  getInitialCharacter: dispatch(addCharacter())
-}
+const mapDispatchToProps = (dispatch)=> {
```

Comment:
probs a bit of es6 refactoring if you want

```
const mapDispatchToProps = dispatch => ({
  getInitialCharacter: () => dispatch(addCharacter())
})
```

Reply:
I am using a linter it does it for me
I can change lint rules if u wish